.gitattributes

```diff
@@ -6,9 +6,12 @@
 # tests
 .codecov.yml export-ignore
 .php-cs-fixer.dist.php export-ignore
+.phpbench export-ignore
```

Comment:
Ah it's in there because I think eventually it shouldn't be in there `.gitignore`.

If we start using a baseline to compare against, that baseline data should probably be committed as well. Or we find a way how only the CI caches a baseline.

Reply:
I feel like running this in CI will be difficult as GitHub Actions workers are shared infrastructure, so probably not stable enough to measure performance consistently. I have no experience with performance regression testing though, so no idea how other projects do it.

Reply:
About the baseline: Doesn't it depend on the system a lot? If I have an environment that's twice as fast as another, the baseline cannot be compared. So wouldn't we need to run both runs (before and after) on the same machine right after another? Which would mean that committing the baseline wouldn't help us.

Reply:
True. Could also make the CI runs really long if we add many performance tests. I guess the question then is whether we'd want to commit (and update) a baseline as part of the repo. Or ignore that folder and everyone creates their own baseline with the main branch and compares their changed code against that baseline locally.

Reply:
Valid point about the system.

Reply:
Think our comments have crossed paths.

About the workflow: Maybe we incorporate it into the release workflow and create an issue for the next release if we find regressions? Would be helpful to see how other projects do it.

Reply:
Indeed. Let's try that once we have the first tests added.

I can already see that we'll have quite a few trial - learn more - refactor phases until we arrive at a state with tests that really makes sense.